Miniforge3/mambaforge_deprecation.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ "$GITHUB_ACTIONS" == "1" ]]; then
```

Comment:
Github sets this to "true" AFAIK. https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Oops, good catch. Thanks!